### DISCUSSION TITLE
federated login with github|apple|google|twitter

### DISCUSSION THREAD

**Discussion by dch** [authelia/authelia#4471] [Q&A] [answered]
Does Authelia support allowing users to authenticate via an
external OAuth2 provider, or similar functionality? Like a 
"login with GitHub" sort of function?

I should add I've read https://www.authelia.com/roadmap/active/openid-connect
and this seems like it *might* do this in future, but it's
not clear to me if that's the intended use case.

**Comment by james-d-elliott** [comment, marked as answer]
Currently the roadmap for OpenID Connect 1.0 is as a provider, not a client. Once we get most of the main provider features sorted we'll look at configuring it as a client.

**Comment by james-d-elliott** [comment]
Tracking issue https://github.com/authelia/authelia/issues/2728
